src/main/java/uk/gov/hmcts/reform/sscs/util/SscsUtil.java

```diff
@@ -43,10 +47,24 @@ public static PreSubmitCallbackResponse<SscsCaseData> processPostponementRequest
             return response;
         }
 
+        DateTimeFormatter dateTimeFormatter = DateTimeFormatter.ofPattern("dd/MM/yyyy");
```

Comment:
I'd pull this out into a constant

Reply:
Agreed, but we also should have one constant for the whole application. It's outside the scope of the current ticket, so I'll make a judgment call to leave it for now.
I'm thinking to set up some kind of "tech debt day" periodically to go through these things.